fe/fe-core/src/main/java/com/starrocks/connector/elasticsearch/external/EsRestClient.java

```diff
@@ -284,4 +295,82 @@ private static SSLSocketFactory createSSLSocketFactory() {
         }
         return ssfFactory;
     }
+
+    /**
+     * response
+     [
+     {
+     "index": ".kibana_1"
+     },
+     {
+     "index": ".opendistro_security"
+     },
+     {
+     "index": "kibana_sample_data_ecommerce"
+     },
+     {
+     "index": "kibana_sample_data_ecommerce_2"
+     },
+     {
+     "index": "kibana_sample_data_flights"
+     },
+     {
+     "index": "kibana_sample_data_logs"
+     }
+     ]
+     * indices are same as table
+     * @return
+     */
+    public Set<String> getIndices() {
+        String response = execute("_cat/indices?h=index&format=json&s=index:asc");
```

Comment:
I think we should add `health` parameter, filter only valid indices.
[https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html#cat-indices-api-query-params](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html#cat-indices-api-query-params)

Reply:
> I think we should add `health` parameter, filter only valid indices. https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html#cat-indices-api-query-params
here provide es all tables means that behavior is same as es？

Reply:
it won't lead  be to  crashed. just empty data